tutorials/hist/NormalizeHistogram.C

```diff
@@ -0,0 +1,73 @@
+/// \file
+/// \ingroup tutorial_hist
+/// \notebook -js
+/// Normalizing a Histogram
+///
+/// Image produced by `.x Normalizing_Histogram.C`
+/// 2 different methods of normalizing Histograms
+/// are shown, each with the original Histogram
+/// next to the normalized one.
+/// \macro_image
+/// \macro_code
+///
+/// The methods are:
+/// 1:
+/// ```
+/// hist2->Scale(1. / hist2->Integral());
+///```
+/// 2:
+/// ```
+///  hist3->Scale(1. / hist3->Integral("width"));
```

Comment:
Can you explain the different of the two methods, here?

The first is simply normalizing by dividing by the bin content; the second is also taking the bin width into account, i.e. normalizing by the sum of the bin's content times their respective widths. This is especially important for variable-sized bins, but also for giving a physical meaning to the norm, such as "20.2 events / GeV" for an x-axis unit of GeV.

Reply:
Actually - it's better to explain this below, where the normalization actually happens. Here you can just remove the part "the methods are" because it's a repetition of the actual code below.

Reply:
Okay, I'll remove these lines and put the description where the actual methods are.